Guide/deployment.markdown

```diff
@@ -12,6 +12,35 @@ AWS EC2 is a good choice for deploying IHP in a professional setup.
 
 ### AWS infrastructure preparation
 
+#### Creating infrastructure with Terraform
+
+The EC2 instance, RDS database, VPS, subnets, security groups, etc, can be setup automatically using [Terraform](https://www.terraform.io/).
+
+1. Install terraform
+1. Setup AWS credentials in .aws/config and .aws/credentials
```

Comment:
```suggestion
2. Setup AWS credentials in `.aws/config` and `.aws/credentials`
```

Reply:
I usually use only 1. since then you get the correct numbering for free in markdown, but I see that other parts of this documentation uses explicit numbering.